src/command_modules/azure-cli-acs/azure/cli/command_modules/acs/custom.py

```diff
@@ -838,15 +838,15 @@ def store_acs_service_principal(subscription_id, client_secret, service_principa
         json.dump(fullConfig, spFile)
 
 
-def load_acs_service_principal(subscription_id, config_path=os.path.join(get_config_dir(),
-                                                                         'acsServicePrincipal.json')):
-    config = load_acs_service_principals(config_path)
+def load_acs_service_principal(subscription_id, file_name='acsServicePrincipal.json'):
+    config_path = os.path.join(get_config_dir(), file_name)
+    config = load_service_principals(config_path)
     if not config:
         return None
     return config.get(subscription_id)
 
 
-def load_acs_service_principals(config_path):
+def load_service_principals(config_path):
```

Comment:
`load_service_principal` ?

Reply:
It could be multiple stored in one file. One per sub_id

Reply:
ok